### PR TITLE
📚 Add README details

### PR DESCRIPTION
**Pull request by ricardoerl**
#11 #7 
- Add install instructions
- Add development instructions
- Add contributing instructions
- Add links section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,45 @@
...
+yarn
 ```
-$ git clone git@github.com:horchatajs/website.git
-$ cd website
-$ yarn
-$ npm run develop
+
+## Desarrollo
+
+> Antes de iniciar, tenés que cambiar el nombre del archivo `.env.example` a `.env` y colocar tu API Key de Meetup (ver prerrequisitos).
```
[the remark this answers is not in the document]
Comment (ricardoerl) [reply README.md]: Actualizado.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,45 @@
-# Website 🔗
+# Website
 
-Sitio web oficial de la comunidad HorchataJS.
+> Sitio web oficial de la comunidad HorchataJS
 
-## Desarrollar local
+El sitio de HorchataJS es un proyecto de código abierto que tiene como objetivo ser una herramienta para promocionar y compartir el contenido de la comunidad. El sitio está hecho con [Gatsby](https://www.gatsbyjs.org/).
 
+## Prerrequisitos
```
[the remark this answers is not in the document]
Comment (ricardoerl) [reply README.md]: Actualizado

File: README.md [hunk 3]
```diff
@@ -1,16 +1,45 @@
-# Website 🔗
+# Website
 
-Sitio web oficial de la comunidad HorchataJS.
+> Sitio web oficial de la comunidad HorchataJS
 
-## Desarrollar local
+El sitio de HorchataJS es un proyecto de código abierto que tiene como objetivo ser una herramienta para promocionar y compartir el contenido de la comunidad. El sitio está hecho con [Gatsby](https://www.gatsbyjs.org/).
```
[the remark this answers is not in the document]
Comment (ricardoerl) [reply README.md]: Actualizado

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cesaramirez) [reply README.md]: 👍

**Pull request merged**